exercises/lottery_game.py

```diff
@@ -0,0 +1,26 @@
+"""Provided a list of lottery players, and also with 6 random lottery numbers.
+Find out the player with the most correct numbers, and print out their winnings and their name
+Task is to find who matched the most numbers, and print out a string with their name and the amount they won. For this exercise, 
+assume there will only be 1 winner. 
+For example: Jen won 1000. 
+The winnings are calculated with this formula:
+winnings = 100 ** len(numbers_matched) """
+
+import random # built-in python module to generate random numbers
```

Comment:
never add unnecessary comments
```suggestion
import random
```